unfetter-discover/unfetter-ui
# Save new baseline to database once created

Initial functionality of auto save.

Fixes unfetter-discover/unfetter#1086

To Test:

- Create a new baseline, don't save it.
    - Check to see if the baseline exists.
- Create a new baseline, save it.
    - Check to see if it exists and there is not a duplicate.
